examples/notebooks/voltage-regulator/publish_waveforms.ipynb

```diff
@@ -100,69 +174,35 @@
     "\n",
     "# Time in milliseconds\n",
     "time_ms = np.linspace(0, 1000, 500)\n",
+    "step = Step(step_name=\"Initial step\", test_result_id=test_result_id)\n",
+    "step_id = client.create_step(step)\n",
     "\n",
     "# Input voltages to simulate\n",
     "input_voltages = [4.5, 7.0, 9.0, 12.0]\n",
-    "colors = ['blue', 'green', 'orange', 'red']\n",
     "\n",
     "# Simulate each input voltage\n",
-    "vout = []\n",
-    "waveforms = []\n",
-    "for vin, color in zip(input_voltages, colors):\n",
+    "for vin in input_voltages:\n",
     "    waveform_data = simulate_voltage_regulator(vin, time_ms)\n",
-    "    vout.append(waveform_data)\n",
-    "    waveforms.append(\n",
-    "        AnalogWaveform(\n",
+    "    waveform = AnalogWaveform(\n",
     "            sample_count=len(waveform_data),\n",
     "            raw_data=np.array(waveform_data),\n",
     "            timing=Timing.create_with_regular_interval(\n",
     "                ht.timedelta(seconds=1e-3),\n",
     "                ht.datetime.now(timezone.utc)\n",
```

Comment:
My preference is that the `simulate_voltage_regulator` method return an AnalogWaveform (not an array of samples). This matches where a user makes a driver call and returns a waveform with t0, dt, and samples. I would not expect anyone calling the publish waveform method will be creating an AnalogWaveform before publishing.

Reply:
See updated DAQmx example which now reads waveform
https://github.com/ni/nidaqmx-python/blob/master/examples/analog_in/voltage_acq_int_clk_plot_wfm.py
https://github.com/ni/nipanel-python/blob/main/examples/nidaqmx/nidaqmx_continuous_analog_input/nidaqmx_continuous_analog_input.py#L95